src/main/java/core/basesyntax/filework/ReadFromCsvFileImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.filework;
+
+import core.basesyntax.exceptions.ReadFromFileException;
+import core.basesyntax.model.Fruit;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadFromCsvFileImpl implements ReadFromFile {
+    private static final String COMA = ",";
+
+    @Override
+    public void readFromFile(String pathToFile) {
+        String line;
+
+        try (BufferedReader br = new BufferedReader(new FileReader(pathToFile))) {
+            br.readLine();
+            while (true) {
+                if ((line = br.readLine()) == null) {
+                    break;
+                }
+                parseToFruit(line.split(COMA));
+            }
+        } catch (IOException e) {
+            throw new ReadFromFileException("Can't rad from file");
+        }
+    }
+
+    private void parseToFruit(String[] line) {
```

Comment:
Метод parse() повинен лише парсити дані, він не повинен звертатися до сервісів
```suggestion
    private Transaction parseTransaction(String line) {
```